Ingest new packages from dotnet/runtime

JunTaoLuo opened the pull request:
As part of the ingestion process for new extensions packages from dotnet/runtime https://github.com/dotnet/aspnetcore/issues/19254

dougbu commented on eng/Version.Details.xml:
```diff
@@ -1,13 +1,13 @@
-    <Dependency Name="Microsoft.Extensions.Configuration.Json" Version="5.0.0-preview.3.20179.1">
-      <Uri>https://github.com/dotnet/extensions</Uri>
-      <Sha>1d4a35ec9ecb9bcd8fb26150fb6da3deb4aab795</Sha>
+    <Dependency Name="Microsoft.Extensions.Configuration.Json" Version="5.0.0-preview.3-runtime.20179.1" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
```
I'm pretty sure this repo doesn't need CPD attributes for anything except the dotnet/roslyn dependency. @wtgodbe @mmitche is that correct i.e. is there any reason for CPD links between packages coming from the same (dotnet/runtime) repo?

wtgodbe replied:
We still need the CPDs while runtime flows through extensions to this repo (which is still the case). Once we ingest runtime directly we can remove the CPDs

dougbu replied:
Huh? The words `dotnet/extensions` don't appear anywhere in this file; everything comes from arcade, roslyn or runtime.

wtgodbe replied:
Oh, I see, I had this backwards - in that case this PR also depends on adding a 'runtime -> aspnetcore-tooling' subscription in Darc

dougbu replied:
yes

JunTaoLuo replied:
I wasn't planning on updating darc but I suppose I could.

wtgodbe commented on eng/Version.Details.xml:
```diff
@@ -1,58 +1,58 @@
       <Sha>7dbc907fa03eacf4c57f827cb4235d77b7ed4fcd</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.6.0-3.20170.19" CoherentParentDependency="Microsoft.Extensions.Logging">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.6.0-3.20170.19" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
```
dotnet/runtime doesn't ingest roslyn - we'll need to add a sub from roslyn to this repo

JunTaoLuo replied:
If runtime doesn't ingest roslyn then do we need the CPD?

wtgodbe replied:
We don't

JunTaoLuo replied:
removed

JunTaoLuo commented:
I've updated the darc subscriptions.

Removed:
```
https://github.com/dotnet/extensions (.NET Core 5 Dev) ==> 'https://github.com/dotnet/aspnetcore-tooling' ('master')
  - Id: 19ed8901-76aa-497e-fb12-08d794a352c5
  - Update Frequency: EveryBuild
  - Enabled: True
  - Batchable: True
  - Merge Policies:
    Standard
```
Added: 
```
https://github.com/dotnet/roslyn (.NET Core 5 Dev) ==> 'https://github.com/dotnet/aspnetcore-tooling' ('master')
  - Id: 030e923b-f1c4-4be7-e17c-08d7d67aa73d
  - Update Frequency: EveryWeek
  - Enabled: True
  - Batchable: True
  - Merge Policies:
    Standard
https://github.com/dotnet/runtime (.NET Core 5 Dev) ==> 'https://github.com/dotnet/aspnetcore-tooling' ('master')
  - Id: 8391606a-bc3e-4040-8a3c-08d7d6786e01
  - Update Frequency: EveryDay
  - Enabled: True
  - Batchable: True
  - Merge Policies:
    Standard
```
